src/components/Groups/GroupAdd.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div>
+    <h1>Add new group</h1>
```

Comment:
It might look cleaner not to have this big title. The rest of form makes it pretty clear what that part of the UI is for.